### PR TITLE
Fix broken links - OLM Documentation was replaced by doc

### PR DESCRIPTION
**Pull request by camilamacedo86**
**Description of the change:**
The Dir of docs in OLM was changed from Documentation to doc. This change is to fix the broken links. 

**Motivation for the change:**

- https://github.com/operator-framework/operator-sdk/issues/1887

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# Intellij IDEA
+.idea
```
Comment (kevinrizza) [inline .gitignore]: This is noise, please remove this from the pull request.

Comment (camilamacedo86) [reply .gitignore]: It is for do not commit ( ignore ) the files from Golang IDEA as it is done for the other things there. If the dev user IDEA files from IDEA that should not have been trying to be committed.

Comment (kevinrizza) [reply .gitignore]: I do understand what the change to the `.gitignore` file does. It is just unrelated to the commit message, pr header, and purpose of the pull request. IMO a pull request to this repo should only include changes directly related to that pull request. If you would like to suggest an update to the `.gitignore` please open a separate pull request for that purpose.

Comment (camilamacedo86) [reply .gitignore]: Ok I catcher your point. Tks for the clarifications.

Comment (camilamacedo86) [reply .gitignore]: Done ..

**Pull request merged**